Document Instaclick (#157)
hotwired/turbo-site: _source/handbook/02_drive.md

Servers may also respond to form submissions with a [Turbo Streams](streams) message by sending the header `Content-Type: text/vnd.turbo-stream.html` followed by one or more `<turbo-stream>` elements in the response body. This lets you update multiple parts of the page without navigating.

> ## Instaclick

Reviewer: Name of the library is InstantClick, though not sure if we're going with a different one on the docs, or if we should call it "Prefetching" instead.

"InstantClick" might help with SEO for users searching for it.

> Turbo can also speed up network requests by automatically loading links on `mouseenter` or `touchstart` events, and before the user clicks the link. This usually leads to a speed bump of 500-800ms per click navigation.

Reviewer: Adding "perceived" since it's technically more correct:

```suggestion
Turbo can also speed up perceived network requests by automatically loading links on `mouseenter` or `touchstart` events, and before the user clicks the link. This usually leads to a speed bump of 500-800ms per click navigation.
```

> Turbo can also speed up network requests by automatically loading links on `mouseenter` or `touchstart` events, and before the user clicks the link. This usually leads to a speed bump of 500-800ms per click navigation.

Reviewer: Suggestion based on your initial description, might want to take some ideas:

>Turbo can also speed up perceived network requests by automatically prefetching links when users hover over a link before clicking it. This is done on `mouseenter` on desktop and on `touchstart` for mobile devices. This usually leads to a speed bump of 500-800ms per click navigation, since the page will already be there when they click.

To enable this behavior add this meta tag to your page:

```html
<meta name="turbo-prefetch" content="true">
```

> To avoid prefetching links that the user is briefly hovering Turbo waits 100ms after the user enters the link before prefetching it. But you may want to disable the prefetching behavior on certain links leading to pages with expensive rendering.

Reviewer: Adding a comma, changing "enter" to "hover" and separating the ideas into 2 paragraphs:

```suggestion
To avoid prefetching links that the user is briefly hovering, Turbo waits 100ms after the user hovers over the link before prefetching it.

You may also want to disable the prefetching behavior on certain links leading to pages with expensive rendering.
```
